style/javascript.md

```diff
@@ -0,0 +1,11 @@
+# Javascript
+
+Including NodeJS and client side scripting.
+
+## [StandardJS](http://standardjs.com/)
```

Comment:
Either way is short and concise.

I think the links highlight that we are just letting standardjs control the linting, so we don't have to worry about configuration.